DEHPCommon/HubController/HubController.cs

```diff
@@ -392,39 +392,92 @@ public IEnumerable<NestedElement> GetNestedElementTree(Option option, DomainOfEx
         /// <param name="iteration"></param>
         /// <param name="domain"></param>
         public async Task Upload(File file = null, Iteration iteration = null, DomainOfExpertise domain = null)
+        {
+            await Upload(null, file, iteration, domain);
+        }
+
+        /// <summary>
+        /// Upload one file to the <see cref="DomainFileStore"/> of the specified domain or of the active domain
+        /// </summary>
+        /// <param name="filePath"></param>
```

Comment:
Could you add parameters documentation please?

Reply:
Yes